pkg/render/logstorage/linseed/linseed.go

```diff
@@ -183,6 +186,17 @@ func (e linseed) linseedDeployment() *appsv1.Deployment {
 
 		// Configuration for connection to Elasticsearch.
 		{Name: "LINSEED_ELASTIC_ENDPOINT", Value: ElasticsearchHTTPSEndpoint},
+		{Name: "LINSEED_ELASTIC_USERNAME", Value: e.cfg.EsAdminUserName},
```

Comment:
I believe we have some common logic for handling ES connection vars. 

e.g., https://github.com/tigera/operator/blob/ddbaa22cd1b351564e952b851bc1b6b5d0aeae0d/pkg/render/compliance.go#L391

Full thing here: https://github.com/tigera/operator/blob/ddbaa22cd1b351564e952b851bc1b6b5d0aeae0d/pkg/render/common/elasticsearch/decorator.go

Reply:
Sure. I think we are not using the prefix of the component to set those variables, so I might need to make some changes in Linseed for this.

Reply:
@caseydavenport I see that this is not setting some environments as I would expect them.
 - ELASTIC_HOST points to es-gateway 
 - ELASTIC_PASSWORD is using the same secret name as the one given as parameter. For the moment, we want user `elastic` and password from `tigera-secure-es-elastic-user`